Patica Andreea/Lab5/p3.9/main.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int calculatePowerAndDivisor (int number, int *divisor)
```

Comment:
Interesting thought here, with using `divisor` as an actual output variable and also returning the power!

It is very important that we talk about this during the lab - it's something of a more general concern, so remind me about it!